_locales/es/messages.json

```diff
@@ -0,0 +1,127 @@
+{
+	"name":
+	{
+		"message": "Pestañas guardadas",
+		"description": "Extension name. Displayed in the manifest and pane header"
+	},
+	"description":
+	{
+		"message": "Classic Microsoft Edge \"Tabs Aside\" feature",
+		"description": "Extension description"
+	},
+	"author":
+	{
+		"message": "Michael \"XFox\" Gordeev",
+		"description": "Author name"
+	},
+	"options":
+	{
+		"message": "Opciones",
+		"description": "Alternative text for options button in the pane"
+	},
+	"closePanel":
+	{
+		"message": "Cerrar",
+		"description": "Alternative text for close panel button"
+	},
+	"loadOnRestore":
+	{
+		"message": "Cargas pestañas al restaurar",
+		"description": "Label for option"
+	},
+	"showDeleteDialog":
+	{
+		"message": "Mostrar diálogo de confirmacion antes de eliminar un item",
+		"description": "Label for option"
+	},
+	"swapIconAction":
+	{
+		"message": "Guardar pestañas al apretar el icono de la extensión (%TOGGLE_SHORTCUT% o click-derecho para abrir el panel)",
+		"description": "Label for option"
+	},
+	"github":
+	{
+		"message": "Visitar la pagina de GitHub",
+		"description": "Link title"
+	},
+	"contributors":
+	{
+		"message": "Contribuidores del proyecto",
+		"description": "Link title"
+	},
+	"feedback":
+	{
+		"message": "Deja un comentario",
+		"description": "Link title"
+	},
+	"buyMeACoffee":
+	{
+		"message": "¡Comprame un cafe!",
+		"description": "Link title"
+	},
+	"credits":
+	{
+		"message": "Desarrollado por Michael 'XFox' Gordeev",
+		"description": "Options menu credits"
+	},
+	"setAside":
+	{
+		"message": "Guardas las pestañas actuales",
+		"description": "Save collection action name. Used in the pane, extension context menu and manifest shortcuts"
+	},
+	"nothingSaved":
+	{
+		"message": "No tienes pestañas guardadas",
+		"description": "Placeholder for empty pane"
+	},
+	"removeTab":
+	{
+		"message": "Eliminar pestañas de colección",
```

Comment:
Spanish-English translation of the string seems a bit off. If says "Remove collection tabs" and according to the GT it missing "la" article. So I guess it should be "Eliminar pestañas de la colección"

Reply:
Both are grammatically correct, is down to preference. With the "la" article it sounds more formal than without it.